File: OPENAPI-REST-API/swagger-client/dart/lib/model/domain_all_info_attributes_contact_set.dart
part of swagger.api;

class DomainAllInfoAttributesContactSet {
  
  DomainOwnerContact owner = null;

  DomainAdminContact admin = null;

  DomainTechContact tech = null;

  DomainAllInfoAttributesContactSet();

  @override
  String toString() {
    return 'DomainAllInfoAttributesContactSet[owner=$owner, admin=$admin, tech=$tech, ]';
  }

  DomainAllInfoAttributesContactSet.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    owner = new DomainOwnerContact.fromJson(json['owner']);
    admin = new DomainAdminContact.fromJson(json['admin']);
    tech = new DomainTechContact.fromJson(json['tech']);
  }

  Map<String, dynamic> toJson() {
    return {
      'owner': owner,
      'admin': admin,
      'tech': tech
     };
  }

  static List<DomainAllInfoAttributesContactSet> listFromJson(List<dynamic> json) {
    return json == null ? new List<DomainAllInfoAttributesContactSet>() : json.map((value) => new DomainAllInfoAttributesContactSet.fromJson(value)).toList();
  }

  static Map<String, DomainAllInfoAttributesContactSet> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, DomainAllInfoAttributesContactSet>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new DomainAllInfoAttributesContactSet.fromJson(value));
    }
    return map;
  }
}

File: OPENAPI-REST-API/swagger-client/csharp-dotnet2/src/main/CsharpDotNet2/myadmin-client-csharp-dotnet2/Model/DomainAllInfoAttributesContactSet.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class DomainAllInfoAttributesContactSet {
    /// <summary>
    /// Gets or Sets Owner
    /// </summary>
    [DataMember(Name="owner", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "owner")]
    public DomainOwnerContact Owner { get; set; }

    /// <summary>
    /// Gets or Sets Admin
    /// </summary>
    [DataMember(Name="admin", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "admin")]
    public DomainAdminContact Admin { get; set; }

    /// <summary>
    /// Gets or Sets Tech
    /// </summary>
    [DataMember(Name="tech", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "tech")]
    public DomainTechContact Tech { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class DomainAllInfoAttributesContactSet {\n");
      sb.Append("  Owner: ").Append(Owner).Append("\n");
      sb.Append("  Admin: ").Append(Admin).Append("\n");
      sb.Append("  Tech: ").Append(Tech).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}
